go.mod

```diff
@@ -716,4 +718,4 @@ exclude (
 
 replace github.com/github/smimesign => github.com/grafana/smimesign v0.2.1-0.20220408144937-2a5adf3481d3
 
-// replace github.com/grafana/phlare/ebpf => ../phlare/ebpf
+// replace github.com/grafana/pyroscope/ebpf => ../pyroscope/ebpf
```

Comment:
Do we need this comment?

Reply:
nope, it is just useful for development without publishing locally. will remove it to not confuse.